-code-checks

ERROR: Build errors found during clang-tidy run.
```
GeneratorInterface/RivetInterface/plugins/HTXSRivetProducer.cc:40:13: error: use of undeclared identifier 'nullprt'; did you mean 'nullptr'? [clang-diagnostic-error]
    _HTXS = nullprt;
            ^~~~~~~
            nullptr
GeneratorInterface/RivetInterface/plugins/HTXSRivetProducer.cc:41:10: error: no viable overloaded '=' [clang-diagnostic-error]
    cat_ = nullptr;
         ^
/cvmfs/cms-ib.cern.ch/nweek-02646/slc7_amd64_gcc820/cms/cmssw-patch/CMSSW_11_2_X_2020-09-15-1100/src/SimDataFormats/HTXS/interface/HiggsTemplateCrossSections.h:514:10: note: candidate function (the implicit move assignment operator) not viable: no known conversion from 'nullptr_t' to 'HTXS::HiggsClassification' for 1st argument
--
gmake: *** [config/SCRAM/GMake/Makefile.coderules:128: code-checks] Error 2
gmake: *** [There are compilation/build errors. Please see the detail log above.] Error 2
```

--- a/GeneratorInterface/RivetInterface/plugins/HTXSRivetProducer.cc
+++ b/GeneratorInterface/RivetInterface/plugins/HTXSRivetProducer.cc
@@ -37,7 +37,8 @@ public:
     usesResource("Rivet");
     _prodMode = cfg.getParameter<string>("ProductionMode");
     m_HiggsProdMode = HTXS::UNKNOWN;
-
+    _HTXS = nullptr;
+    _analysisHandler = nullptr;
     produces<HTXS::HiggsClassification>("HiggsClassification").setBranchAlias("HiggsClassification");
   }
 

